README.rst

```diff
@@ -16,6 +9,46 @@ Purpose
 
 Redis Streams implementation for the Open edX event bus.
 
+Overview
+********
+This package implements an event bus for Open EdX using Redis streams.
+
+The event bus acts as a broker between services publishing events and other services that consume these events.
+
+This package contains both the publishing code, which processes events into messages to send to the stream, and the consumer code,
+which polls the stream using a `while True` loop in order to turn messages back into event data to be emitted.
+The actual Redis host is configurable.
+
+The repository works together with the openedx/openedx-events repository to make the fully functional event bus.
+
+Documentation
+*************
+
+To use this implementation of the Event Bus with openedx-events, you'll need to ensure that below the following Django settings are set::
+
+    # redis connection url
+    EVENT_BUS_REDIS_CONNECTION_URL: ...
+    EVENT_BUS_TOPIC_PREFIX: ...
+
+    # Required, on the producing side only:
```

Comment:
Can you give a quick explanation of what these do so people understand why they're setting them?